src/knockout.contextmenu.js

```diff
@@ -1,15 +1,21 @@
-(function (undefined) {
-'use strict';
-
-if (typeof ko !== undefined + '' && typeof document !== undefined + '') {
-  bindContextMenu(ko, document);
-}
-
-if (typeof module !== undefined + '' && module.exports) {
-  module.exports = bindContextMenu;
-}
-
-function bindContextMenu(ko, document) {
+/* knockout.contextmenu v0.4.4
+   Nicolás Escalante - nlante@gmail.com
+   Issues: https://github.com/nescalante/knockout.contextmenu/issues
+   License: MIT */
+/*globals define,module,exports*/
+(function (factory) {
+  'use strict';
+  if (define !== 'undefined' && typeof define === 'function' && define.amd) {
+    // AMD. Register as an anonymous module.
+    define(['knockout'], factory);
+  } else if (typeof module !== undefined + '' && module.exports) {
+    factory(module.exports || exports);  // module.exports is for Node.js
+  } else {
+    // Browser globals
+    factory(ko);
+  }
+}(function (ko) {
```

Comment:
adding an `undefined` second property here will save a couple of bytes on the bundle